lua/packer.lua

```diff
@@ -784,6 +790,22 @@ packer.loader_complete = function(lead, _, _)
   return completion_list
 end
 
+--- Completion for listing snapshots in `snapshot_path`
+--- Intended to provide completion for PackerRollback command
+--- TODO: using vim.fn.readdir to get entries the snapshot directory
+packer.snapshot_complete = function(lead, _, _)
+  local completion_list = {}
+  local res = io.popen('ls ' .. config.snapshot_path, 'r')
```

Comment:
Minor nitpick: we'd generally prefer to use the `vim.loop.fs_*` functions rather than running `ls` directly. This helps `packer` be OS-independent, and may help with performance.